Patch 1

jayhuggins opened the pull request:
My changes to /setup.py and /ckanext/datajson/__init__.py fixed the errors I was seeing using paster, and seem to have removed the requirement for the "import ckanext" hack in apache.wsgi.

zr2d2 commented:
Jay, what problems are you seeing with paste commands? I've been using some of the commands in production without a problem. Also, you made changes to the `v1.1` branch, could you change the request to point into the correct upstream branch? If you're having trouble with the development branch, perhaps the master branch would work better, as the v1.1 branch is for generating a v1.1 schema

jayhuggins commented:
Hi Zach, I reverted my changes and moved them to master.  The comment for /setup.py on master includes the error dump I saw in paster, and as noted this change allowed me to remove the "import ckanext" statement from apache.wsgi.

From: Zach Rowe <notifications@github.com<mailto:notifications@github.com>>
Reply-To: HHS/ckanext-datajson <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, November 24, 2014 8:18 AM
To: HHS/ckanext-datajson <ckanext-datajson@noreply.github.com<mailto:ckanext-datajson@noreply.github.com>>
Cc: Jay Huggins <jay.huggins@nrel.gov<mailto:jay.huggins@nrel.gov>>
Subject: Re: [ckanext-datajson] Patch 1 (#14)

Jay, what problems are you seeing with paste commands? I've been using some of the commands in production without a problem. Also, you made changes to the v1.1 branch, could you change the request to point into the correct upstream branch? If you're having trouble with the development branch, perhaps the master branch would work better, as the v1.1 branch is for generating a v1.1 schema

—
Reply to this email directly or view it on GitHubhttps://github.com/HHS/ckanext-datajson/pull/14#issuecomment-64208088.

zr2d2 commented:
Jay, my intent was actually the opposite. Since you included commits from the v1.1 branch, I wanted you to change the base branch of the merge, so that I could evaluate your changes irrespective of my own.
